quick-start/app/build.gradle.kts

```diff
@@ -0,0 +1,98 @@
+import java.util.Properties
+
+plugins {
+    alias(libs.plugins.androidApplication)
+    alias(libs.plugins.jetbrainsKotlinAndroid)
+}
+
+android {
+    namespace = "com.amazon.androidquickstartapp"
+    compileSdk = 34
+    val customConfig = Properties()
+    customConfig.load(project.rootProject.file("custom.properties").inputStream())
+    defaultConfig {
+        applicationId = "com.amazon.androidquickstartapp"
+        minSdk = 24
```

Comment:
Can we change this value to 21, so that it aligns with our other libraries?

Reply:
Updated